### DISCUSSION TITLE
How to enter github student pack

### DISCUSSION THREAD

**Discussion by sadmanhossen** [community/community#139270] [GitHub Education] [answered]
### Hi everyone,

I got github student developer pack approval three months prior but I didn't used it on that time due to personal crisis. But now I couldn't find any option to enter this pack.
Can anyone tell me how could I enter ?

**Comment by davevad93** [comment, marked as answer]
Hi @sadmanhossen ,

Here's the [pack offers page](https://education.github.com/pack), there's a complete list of everything available with the `Student Developer Pack`. Scroll down to the “All Offers” section, there are many interesting educational tools and platforms available for free.

**Reply by sadmanhossen** [reply]
Thanks Brother🙏🏻

**Reply by davevad93** [reply]
You're welcome @sadmanhossen 👍
